.github/workflows/test.yaml

```diff
@@ -0,0 +1,98 @@
+name: test
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+
+jobs:
+  test:
+    name: Run tests and collect coverage (python)
```

Comment:
Rename so that truncated name is more useful
![CleanShot 2024-12-13 at 14 35 07@2x](https://github.com/user-attachments/assets/f9384dc8-e62f-4390-add3-362df86627fd)

Reply:
This is how it appears with changes now, not sure if that's what you want?
![image](https://github.com/user-attachments/assets/77a08e66-5174-4a37-b7b7-8db27a1783e1)